### PR TITLE
[DOCFIX] Improve Alibaba ACK doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
Various minor doc improvements on the grammar/expression/format etc.

### CODE REVIEW

File: docs/en/cloud/Alibaba-Cloud-ACK.md [hunk 1]
```diff
@@ -106,40 +106,49 @@ After the download is complete, unzip the package and set env "`SPARK_HOME`":
...
  {% endcollapsible %}
 
  {% collapsible Upload files to Alluxio %}
-As mentioned at the beginning of the document: This experiment is to submit a spark job to k8s and the goal of the spark job is to count the number of occurrences of each word for a certain file. Now you need to upload the file to the alluxio storage. Here, for convenience, you can directly upload the file `/opt/alluxio-{{site.ALLUXIO_VERSION_STRING}}/LICENSE` in the alluxio master (the file path may be slightly different due to the alluxio version) to alluxio.
+As mentioned at the beginning: In this experiment we will submit a Spark job to K8s. 
+The Spark job will perform a word count calculation on a certain file. 
+Before we kick off the Spark job, we need to upload the file to the Alluxio storage. 
+Here, for convenience, we directly upload the file `/opt/alluxio-{{site.ALLUXIO_VERSION_STRING}}/LICENSE` from the Alluxio master node (the file path may be slightly different due to the Alluxio version) to Alluxio.
```
Comment (LuQQiu) [inline docs/en/cloud/Alibaba-Cloud-ACK.md:137]: from alluxio master node to alluxio?

File: docs/en/cloud/Alibaba-Cloud-ACK.md [hunk 2]
```diff
@@ -148,9 +157,9 @@ Containing the following blocks:
 BlockInfo{id=16777216, length=27040, locations=[BlockLocation{workerId=8217561227881498090, address=WorkerNetAddress{host=192.168.8.17, containerHost=, rpcPort=29999, dataPort=29999, webPort=30000, domainSocketPath=, tieredIdentity=TieredIdentity (node=192.168.8.17, rack=null)}, tierAlias=MEM, mediumType=MEM}]}
 ```
 
-As shown, this `LICENSE` file has only one block whose id is 16777216,  placed on the k8s node of **192.168.8.17**.
+As shown, this `LICENSE` file has only one block whose id is 16777216,  placed on the K8s node **192.168.8.17**.
```
Comment (LuQQiu) [inline docs/en/cloud/Alibaba-Cloud-ACK.md:160]: better to use backstick around 192.168.8.17

File: docs/en/cloud/Alibaba-Cloud-ACK.md [hunk 3]
```diff
@@ -148,9 +157,9 @@ Containing the following blocks:
...
 
-We use `kubectl` to find out that the node name is **cn-beijing.192.168.8.17**:
+We use `kubectl` to identify that the node name is **cn-beijing.192.168.8.17**:
```
Comment (LuQQiu) [inline docs/en/cloud/Alibaba-Cloud-ACK.md:162]: may be better to use backstick around cn-beijing.192.168.8.17

File: docs/en/cloud/Alibaba-Cloud-ACK.md [hunk 4]
```diff
@@ -165,13 +174,20 @@ cn-beijing.192.168.8.17  192.168.8.17
   {% endcollapsible %}
 
   {% collapsible Submit Spark job %}
-The following steps will submit a spark job to the k8s cluster. The job is mainly to count the number of occurrences of each word in the `/LICENSE` file in alluxio.
+The following steps will submit a Spark job to the K8s cluster. 
+The job is mainly to count the number of occurrences of each word in the `/LICENSE` file in Alluxio.
 
-In step 5.3, we obtained that the blocks contained in the LICENSE file are all on the node cn-beijing.192.168.8.17. In this experiment, we specified the node selector to let the spark driver and spark executor run on the node cn-beijing. 192.168.8.17, verify that the communication between spark executor and alluxio worker is completed through the domain socket when alluxio's short-circuit function is turned on.
+In the previous step, we see that the blocks contained in the LICENSE file are all on the node `cn-beijing.192.168.8.17`.
```
Comment (LuQQiu) [inline docs/en/cloud/Alibaba-Cloud-ACK.md:180]: `LICENSE`

**Review by LuQQiu — COMMENTED** [review]
Thanks! Overall LGTM, left some minor comments

File: docs/en/cloud/Alibaba-Cloud-ACK.md (hunk 1 above)
Comment (jiacheliu3) [reply docs/en/cloud/Alibaba-Cloud-ACK.md:137]: What about `from the Alluxio master node to the Alluxio namespace`? It means `copyFromLocal` from the node into Alluxio.

**Comment by jiacheliu3** [comment]
alluxio-bot, test this please

**Review by LuQQiu — APPROVED** [review]

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**